### DISCUSSION TITLE
Add optional MITRE ATT&CK classification to findings

### DISCUSSION THREAD

**Discussion by MichaelGrafnetter** [Syslifters/sysreptor#306] [General]
Many infrastructure pentesters use the [MITRE ATT&CK](https://attack.mitre.org/matrices/enterprise/) framework. The ability to map a vulnerability/attack step to a technique ID, e.g. [Account Manipulation: SSH Authorized Keys](https://attack.mitre.org/techniques/T1098/004/), would be great.

**Comment by aronmolnar** [comment]
What you mean is that we provide a predefined field holding all techniques and tactics hierarchically structured?  
Similar to what exists with the CWE field?

![image](https://github.com/user-attachments/assets/cae88224-64a3-4582-a95b-d1f352b6c7ea)

This shouldn't be too hard to implement. What we must not forget is to include the [license/terms of use](https://attack.mitre.org/resources/legal-and-branding/terms-of-use/).

**Comment by MichaelGrafnetter** [comment]
Yep, that's the idea. Both classifications are actually managed by the same MITRE organization. In the PDF report, the IDs should be hyperlinked to the source material.
